doc/refman/RefMan-tac.tex

```diff
@@ -1320,6 +1356,14 @@ \subsection{\tt assert ( {\ident} :\ {\form} )}
   destructed. If the \texttt{as} {\intropattern} clause generates more
   than one subgoal, {\tac} is applied to all of them.
 
+\item \texttt{eenough ({\ident} :\ {\form}) by {\tac}}\tacindex{eenough}\tacindex{eenough as}\tacindex{eenough by}\\
+ \texttt{eenough {\form} by {\tac}}\tacindex{enough by}\\
+ \texttt{eenough {\form} as {\intropattern} by {\tac}}
+
+  While the different variants of \texttt{eenough} expect that no
+  existential variables are generated by the tactic, \texttt{eenough}
+  removes this constraint.
```

Comment:
@maximedenes Judging by the rest of the documentation, this is a typo and Hugo meant to say `enough` instead of `eenough` in the first occurrence of the two.